hr_recruitment_security/tests/test_access_rights.py

```diff
@@ -23,7 +18,7 @@ def setUpClass(cls):
                 "login": "hr_officer_own_documents",
                 "email": "hr_officer_own_documents@example.com",
                 "groups_id": [
-                    (6, 0, [cls.env.ref("hr_recruitment_security.%s" % group_name).id])
+                    (6, 0, [cls.env.ref("hr_recruitment_security.%s" % group_name).id],)
```

Comment:
Why this extra comma?

Reply:
That was Black's job :D, I already remove it.